src/main/kotlin/com/cout970/magneticraft/tileentity/heat/TileBrickFurnace.kt

```diff
@@ -22,8 +22,8 @@ class TileBrickFurnace : TileHeatBase() {
             specificHeat = COPPER_HEAT_CAPACITY * 3,
             maxHeat = (COPPER_HEAT_CAPACITY * 3 * COPPER_MELTING_POINT).toLong(),
             conductivity = DEFAULT_CONDUCTIVITY,
-            worldGetter = this::getWorld,
-            posGetter = this::getPos)
+            worldGetter = { this.world },
+            posGetter = { this.getPos() })
```

Comment:
You need to refresh the gradle project to download the lastest version of kotlin and be able to use 'this::getWorld' instead of '{ this.world }'